EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/Router/MessageRouter.cs

```diff
@@ -0,0 +1,69 @@
+using Discord.WebSocket;
+using HK47.MessageHandlers.Interfaces;
+using HK47.Services;
+
+namespace HK47.Router;
+
+public class MessageRouter(IDataService dataService, ISteamMessageHandler steamHandler, IGeneralMessageHandler generalHandler, IMessageService messageService) : IMessageRouter
+{
+    public async Task ProcessMessage(SocketMessage message)
+    {
+        try
+        {
+            Console.WriteLine("Processing Message...");
+            if (!message.Content.StartsWith('.'))
+            {
+                // Ignore message.
+                Console.WriteLine("Ignoring Message....");
+                return;
+            }
+            await messageService.ValidateLength(message.Content);
+
+            // Removes the '.' prefix to indicate a bot command.
+            var messageCommand = message.Content.Remove(0, 1);
+            var messageChannel = message.Channel;
+            await RouteCommand(messageCommand);
+
+            Console.WriteLine(messageCommand + "  command processed. ");
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error Processing Command");
```

Comment:
Do we potentially need to have some Discord output if issue processing message (forget if handeled elsewhere, will close if we are informing the user)

Reply:
I think we will add this in the next PR.
At the moment I think just getting this in Main is for the best